### DISCUSSION TITLE
Search Filters / Search Results Help

### DISCUSSION THREAD

**Discussion by L23User** [microsoft-search/pnp-modern-search#3297] [Q&A]
Hi Everyone,

A couple of questions to see if what I am trying to do is possible.

1. How do I get the Search results to default to showing results from the first Search Filter?
2. Can all Search Filters stay visible when a filter is selected?
3. Can the collapse and search a value be removed or hidden from the Search Filters Web Part?

Thanks! :)

**Comment by wobba** [comment]
1. You can deep link in the search box redirect for example
2. Not with the current implementation as filters are dynamic based on the results - having state with all/disabled is a PITA to implement
3. You can use a custom filter template and change I believe
